src/formatDate.js

```diff
@@ -9,6 +9,67 @@
  */
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const partsOfOldDate = date.split(fromFormat[fromFormat.length - 1]);
```

Comment:
You are assuming that `fromFormat` always ends with the separator, which is true according to the task description. However, it would be more robust to explicitly handle the separator. Consider defining a variable for the separator to avoid confusion and potential errors in the future.